### ISSUE TITLE
About the DiodeMix optimizer

### ISSUE THREAD

**Issue by hanhanpp** [GreenBitAI/green-bit-llm#19]
The DiodeMix provide different update strategies for different bit parameters. But I find that only 1-bit parameter's update method is spcified, others, e.g. 8-bit parameter, are same as Adam optimzer.  Is that right?

**Comment by yanghaojin** [comment]
Correct, the gradient update mechanism is modified based on the AdamW optimizer, and supports the alternating update of quantization-related parameters such as zeros and scales and weight. The update for scales will be released in the upcoming version.
